src/main/kotlin/com/devh/vitstore/controller/api/v1/sample/SampleUserController.kt

```diff
@@ -0,0 +1,64 @@
+package com.devh.vitstore.controller.api.v1.sample
+
+import com.devh.vitstore.model.sample.RegistrationResDto
+import com.devh.vitstore.model.sample.UserDto
+import io.swagger.annotations.*
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/api/v1/sample/user")
+class SampleUserController {
+    @PostMapping("/register")
+    @ResponseStatus(HttpStatus.CREATED)
+    @ApiResponses(
+        value = [
+            ApiResponse(
+                code = 201,
+                message = "Registered User",
+                examples = Example(
+                    value = [
+                        ExampleProperty(
+                            mediaType = "application/json",
+                            value = "{'active_token': 'example', 'type': 'REGISTRATION'}"
+                        )
+                    ]
+                )
+            ),
+            ApiResponse(
+                code = 400,
+                message = "Bad Request",
+                examples = Example(
+                    value = [
+                        ExampleProperty(
+                            mediaType = "application/json",
+                            value = "{'errors': [{'error_code': '600', 'error_message': 'Email existed'}," +
+                                "{'error_code': '606', 'error_message': 'Missing email or password'}]}"
+                        )
+                    ]
+                )
+            ),
+            ApiResponse(
+                code = 422,
+                message = "Unprocessable Entity",
+                examples = Example(
+                    value = [
+                        ExampleProperty(
+                            mediaType = "application/json",
+                            value = "{'errors': [{'error_code': '601', 'error_message': 'Incorrect email format'}," +
+                                "{'error_code': '607', 'error_message': 'Invalid Birthday'}]}"
+                        )
+                    ]
+                )
+            )
+        ]
+    )
+    @ResponseBody
+    @ApiOperation(value = "This method create a new user", notes = "Document for API 1.1")
```

Comment:
`notes = "Document for API 1.1"` thừa và sai e ak

Reply:
@huorlk-0450 e thấy note vào có thừa gì đâu anh

Reply:
@tupm-1335 nó được mô tả ở ApiInfo rồi nên ko cần ở đây, cùng với đó ver là 1.0 nhể

Reply:
@huorlk-0450 cái này note là để biết API này tương ứng với API 1.1 trong doc mà a. liên quan gì đến version gì đâu